lib/browser-auto-tracking.js

```diff
@@ -196,11 +197,20 @@ export function initAutoTrackingCore(lib) {
     if (options.recordClicks === true) {
       utils.listener('a, a *').on('click', function(e) {
         const el = e.target;
-        const event = {
+        let event = {
           element: helpers.getDomNodeProfile(el),
-          local_time_full: new Date().toISOString()
+          local_time_full: new Date().toISOString(),
         };
 
+        // pointer position tracking
+        if(options.recordClicksPositionPointer === true) {
+          const pointer = {
+            x_position: e.pageX,
+            y_position: e.pageY,
+          }
+          event = {...event, pointer};
+        }
+
```

Comment:
please add information about this config option to the docs related to autocollector